README.md

```diff
@@ -101,7 +101,7 @@ pip install -r requirements.txt
 
 Then run the tests with:
 ```sh
-pytest --log-cli-level=debug
+python3 -m pytest --log-cli-level=debug
```

Comment:
This shouldn't be necessary

Reply:
Without this, it uses the system `pytest` version instead of the one from the virtualenv. If there is no system `pytest` installed, it fails.

Reply:
If you have the virtualenv active and pytest installed inside it then just invoking `pytest` should pick up that one.

Reply:
For some reason, it didn't work for me.